README.md

```diff
@@ -1109,6 +1112,29 @@ databricks labs ucx report-account-compatibility --profile labs-azure-account
 
 [[back to top](#databricks-labs-ucx)]
 
+## `export-assessment` command
+
+```commandline
+databricks labs ucx export-assessment
+```
+The `export-assessment` command exports UCX results to a specified location and prompts for details on the path and report type. If not specified, it defaults to exporting the main results to the current directory as `export_to_zipped_csv.zip`.
```

Comment:
I would prefer that the "specified location" becomes the output zip, instead of the hardcoded `export_to_zipped_csv.zip`. (That could be the default though) See: https://github.com/databrickslabs/lsql/pull/269#discussion_r1756383880

```suggestion
The `export-assessment` command exports UCX results to a specified location and prompts for details on the path and report type. If not specified, it defaults to exporting the main results to the current directory as `export_to_zipped.csv.zip`.
```